smurfs/src/components/App.js

```diff
@@ -1,22 +1,26 @@
-import React, { Component } from 'react';
-import './App.css';
-/*
- to wire this component up you're going to need a few things.
- I'll let you do this part on your own. 
- Just remember, `how do I `connect` my components to redux?`
- `How do I ensure that my component links the state to props?`
- */
-class App extends Component {
-  render() {
-    return (
-      <div className="App">
-        <h1>SMURFS! 2.0 W/ Redux</h1>
-        <div>Welcome to your Redux version of Smurfs!</div>
-        <div>Start inside of your `src/index.js` file!</div>
-        <div>Have fun!</div>
-      </div>
-    );
-  }
-}
+import React, { Fragment } from 'react';
+import { connect } from 'react-redux';
 
-export default App;
+import SmurfForm from '../components/SmurfForm';
+import Smurfs from '../components/Smurfs';
+import GlobalStyle from './GlobalStyle';
+
+const App = props => {
+  return (
+    <Fragment>
+      <GlobalStyle />
+      <SmurfForm />
+      <Smurfs smurfs={props.smurfs} />
+    </Fragment>
+  );
+};
+const mapStateToProps = state => ({
```

Comment:
`fetchingSmurfs` and `error` are unused so you can avoid using `mapStateToProps` and `connect()` here by implementing this logic in `Smurfs.js` instead